Bottom: 51883315e8ca8a0580467ee44e19391d83ba22e8
Top:    c8c37d9c730e634f0b7c7dd1c20835f331c8f6a7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:36 +0000

Bump circle approximation detail level for gerber export


---

diff --git a/src/polygon.h b/src/polygon.h
index 58634f7..90d0d26 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 24 //20 //8 //40
+#define POLY_CIRC_SEGS 40 //24 //20 //8 //40
 #define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
